CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+## Version 0.2.6 (12-28-2022)
+
+### Security
+ - Introduce API wrapper for custom themes and modules to facilitate easier CSRF tokens (#612)
+
+## Bugfixes
+ - Fixed an issue with alias under Apache2 (#626)
+ - Add an additional check for CSFR when php://input is empty (#626)
```

Comment:
I'd suggest this instead:
Better handle `php://input` being empty when checking the CSRF token.